packages/bpk-component-calendar/examples.js

```diff
@@ -349,13 +349,13 @@ const WeekExample = () => {
         <tbody>
           <Week
             {...weekProps}
-            selectionStart={parseDate(`19800611`)}
-            selectionEnd={parseDate(`19800616`)}
+            selectionStart={parseDate(`19800611`, 'yyyyMMdd', new Date())}
+            selectionEnd={parseDate(`19800616`, 'yyyyMMdd', new Date())}
```

Comment:
`parseDate` now requires you to specify how it's formatted, hence the addition of `yyyyMMdd`. The `new Date()` part is a backup to return if it couldn't parse it.